cluster/gce/util.sh

```diff
@@ -868,7 +868,7 @@ function setup-monitoring-firewall {
   done
   if [[ ${grafana_host} != *"<"* ]]; then
```

Comment:
cc @vishh 

I found the title of this PR a title deceptive even if the change is cool, because it's not "fixing" the address, so much as changing it to go through the proxy.

I would remove lines 859-867. We already have "Wait for the monitoring dashboard to be online."